scripts/rewrite_sqlite_embeddings_to_vec.py

```
@@ -0,0 +1,116 @@
+        )
+        conn.execute(
+            f"""
+            INSERT INTO embeddings_new (
+                id, doc_type, cre_id, node_id, embeddings_url,
+                embeddings_content, embedding_model_id, embedding_dim, embedding_vec
+            )
+            SELECT
+                {select_id}, doc_type, cre_id, node_id, {select_url},
+                {select_content}, {select_model}, {select_dim}, embedding_vec
+            FROM embeddings
+            """
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)